quirks/password-rules.json

```diff
@@ -329,6 +329,9 @@
     "portal.edd.ca.gov": {
         "password-rules": "minlength: 8; required: lower; required: upper; required: digit; required: [!#$%&()*@^];"
     },
+    "portals.emblemhealth.com": {
+        "password-rules": "minlength: 8; required: lower; required: upper; required: digit; required: [!#$%&'()*+,./:;<>?@\^_`{|}~];"
```

Comment:
A couple of notes:

* The backslash character in the custom `required` class needs to be escaped (so make it `\\`)
* I see that `[` and `]` are also allowed special characters based on the screenshot. Did you determine through experimentation that they're not actually allowed? Otherwise, please add them, ensuring that the `]` character is at the end next to the final `]`.

Reply:
Done!